v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-many-models/auto-ml-forecasting-backtest-many-models.ipynb

```diff
@@ -492,25 +524,29 @@
    "source": [
     "For many models we need to provide the ManyModelsInferenceParameters object.\n",
     "\n",
-    "#### ManyModelsInferenceParameters arguments\n",
+    "#### ``ManyModelsInferenceParameters`` arguments\n",
     "| Property                           | Description|\n",
     "| :---------------                   | :------------------- |\n",
     "| **partition_column_names**         | List of column names that identifies groups.                                           |\n",
-    "| **target_column_name**             | \\[Optional\\] Column name only if the inference dataset has the target. |\n",
-    "| **time_column_name**               | Column name only if it is timeseries. |\n",
-    "| **many_models_run_id**             | \\[Optional\\] Many models pipeline run id where models were trained. |\n",
+    "| **target_column_name**             | \\[Optional] Column name only if the inference dataset has the target. |\n",
+    "| **time_column_name**               | \\[Optional] Column name only if it is timeseries. |\n",
+    "| **many_models_run_id**             | \\[Optional] Many models run id where models were trained. |\n",
```

Comment:
Please change "Many models run id where models were trained" to "Many models run id used for training".

Reply:
Removing this as this is not available as a parameter, instead customers can use ``train_run_id`` parameter in ``get_many_models_batch_inference_steps``.